.drone.yml

```diff
@@ -4,6 +4,7 @@ workspace:
 
 branches:
   - master
+  - die-jenkins-die
```

Comment:
This got to be dropped

Reply:
Yes of course, I'm still testing everything - will remove before merging obviously.